README.md

```diff
@@ -70,24 +70,39 @@ There are several good examples and tutorials on the web:
 * Google's [introduction to push notifications](https://developers.google.com/web/fundamentals/getting-started/push-notifications/) (as of 03-20-2016, it doesn't mention notifications with payload)
 * you may want to take a look at my own implementation: [sw.js](https://github.com/Minishlink/physbook/blob/2ed8b9a8a217446c9747e9191a50d6312651125d/web/service-worker.js) and [app.js](https://github.com/Minishlink/physbook/blob/d6855ca8f485556ab2ee5c047688fbf745367045/app/Resources/public/js/app.js)
 
-### GCM servers notes (Chrome)
-For compatibility reasons, this library detects if the server is a GCM server and appropriately sends the notification.
+### Authentication
```

Comment:
Would be good to keep this as just GCM.

FCM endpoint is completely compatible with VAPID and web push protocol and requires nothing proprietary.

GCM PushSubscriptions have the old API key requirement